tests/handlers/test_freshmaker.py

```diff
@@ -72,3 +72,29 @@ def test_event_to_building(mock_koji_cs):
     assert build is not None
     assert build.original_nvr == orig_nvr
     assert event.triggered_container_builds.is_connected(build)
+
+
+@mock.patch('koji.ClientSession')
+def test_build_state_change(mock_koji_cs):
```

Comment:
Since the message you are testing is a build transitioning to the done state, that means the build should already exist in Neo4j but is just being updated. Could you please simulate this by populating Neo4j with that container build?

Reply:
This comment is still applicable